install/kubernetes/helm/istio/values.yaml

```diff
@@ -329,6 +329,12 @@ global:
   # the desired values.
   defaultNodeSelector: {}
 
+  # Default node tolerations to be applied to all deployments so that all pods can be 
```

Comment:
@morvencao is tolerate only used in conjunction with taint? If so, can we make it clear in this comment?

Reply:
lgtm otherwise :)

Reply:
done